tracee-ebpf/tracee/events_definitions.go

```diff
@@ -86,17 +87,6 @@ const (
 	MaxUserSpaceEventID
 )
 
-const (
-	NetPacket uint32 = iota
-	DebugNetSecurityBind
-	DebugNetUdpSendmsg
-	DebugNetUdpDisconnect
-	DebugNetUdpDestroySock
-	DebugNetUdpV6DestroySock
-	DebugNetInetSockSetState
-	DebugNetTcpConnect
-)
-
```

Comment:
I agree but because we using them in the `netproto` pkg we have to declare them there too (or to include Tracee to the `netproto` pkg which a bit problematic)